omisego-sdk/src/main/java/co/omisego/omisego/network/ewallet/EWalletClient.kt

```diff
@@ -114,6 +124,7 @@ class EWalletClient {
             eWalletClient.retrofit = Retrofit.Builder().apply {
                 addConverterFactory(OMGConverterFactory.create(gson))
                 addCallAdapterFactory(OMGCallAdapterFactory.create())
+                callbackExecutor(callbackExecutor)
```

Comment:
I think `callbackExecutor(callbackExecutor ?: MainThreadExecutor)` and removing the test above would be easier to follow as long as we don't do anything else on the executor, but not a big deal

Reply:
Done 😄